datafn/python/datafn/handlers/mutation.py

```
@@ -0,0 +1,844 @@
+            "affectedIds": affected_ids,
+            "deduped": False,
+            "errors": []
+        }
+
+        # Write change tracking
+        if changes:
+            for change in changes:
+                try:
+                    await _write_change(config.db, change, namespace)
+                except Exception:
+                    pass  # Change tracking failure is non-fatal
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)